src/main/scala/beam/sim/BeamServices.scala

```diff
@@ -248,18 +249,6 @@ object BeamServices {
           val rechargeLevel3RateLimitInWatts = Option(line.get("rechargeLevel3RateLimitInWatts")).map(_.toDouble)
           val vehicleCategory = VehicleCategory.fromString(line.get("vehicleCategory"))
 
-          // This is a hack, hope we can fix files soon...
```

Comment:
Just making sure this was explicitly fixed in this PR - and not just a bad merge somewhere?